lib/Common/Memory/CustomHeap.h

```diff
@@ -89,7 +89,8 @@ struct Allocation
         } largeObjectAllocation;
     };
 
-    __field_bcount(size) char* address;
```

Comment:
These annotations help with static analysis tools on Windows and might have other implications to the compiler as well. Would prefer not to outright remove it if possible.

A localized solution would be to check whether this is a linux build and use #ifdef to select your new version.

A better solution would be to define the problematic macros under linux builds, again using #ifdef so that the definition doesn't conflate with the definition already being used on Windows.